### ISSUE TITLE
Vets-api terminal access for Rockwell Rice

### ISSUE THREAD

**Issue by rockwellwindsor-va** [department-of-veterans-affairs/va.gov-team#60529]
### E-QIP Transmittal Date

- [X] Check this box if you have an E-QIP transmittal date.

### Your Name

Rockwell Rice

### Your GitHub handle (terminal access is managed through GitHub)

@rockwellwindsor-va

### Team, Role, and Company of the target individual

Dash, Software Engineer II, Oddball

### Product Manager (PM) name and email

Kayla Watanabe, kayla.watanabe@adhocteam.us

### Product Owner (PO) name and email

Janet Coutinho, janet.coutinho@va.gov

### ArgoCD terminal access requested for the following environments

- [X] dev
- [X] staging
- [X] sandbox
- [X] production (if requesting, justify prod access in 'Additional Notes' section)

### COR Name (leave "None" if you listed an AWS user name above)

Cecila Lee

### Vendor Onboarding Representative (VOR) Name (leave "None" if you listed an AWS user name above)

Oddball - Amber Malcolm

### Access Expiration

ongoing

### Additional Notes

Would like to obtain access to all environments. We have to manually resubmit failed Claim Submissions to EVSS and BGS from within Dash and working with Platform-Support is an additional step that we would like to bypass to troubleshoot and update submissions. It would be best if an individual more aligned with the code were updating and running resubmissions.

You can find me on the VFS Roster

* I have filed AWS access request, https://github.com/department-of-veterans-affairs/va.gov-team/issues/60180, when that gets granted I will update this with the information for that field

eQUIP
<img width="1049" alt="equip" src="https://github.com/department-of-veterans-affairs/va.gov-team/assets/129893414/8bf0fe83-23a9-4e1f-a596-ecca3bae0117">

### User must exist in a roster before vets-api terminal access can be granted

- [x] Search for user on the [VFS Team Roster](https://docs.google.com/spreadsheets/d/11dpCJjhs007uC6CWJI6djy3OAvjB8rHB65m0Yj8HXIw/edit?folder=0ALlyxurHpUilUk9PVA#gid=2042046665)
- [ ] Or search for user on the [Platform Team Roster](https://vfs.atlassian.net/wiki/spaces/AP/pages/1908834623/Platform+Roster)
- [ ] If user is on a VFS team but not in the VFS Team Roster, add the 'NOT YET' label and instruct them to start the [Platform orientation process](https://depo-platform-documentation.scrollhelp.site/getting-started/Platform-Orientation.1877344532.html)
- [ ] If user is on a Platform team but not on the Platform Team Roster in Confluence, add the 'NOT YET' label and instruct them to reach out to their Product Manager to be added
- [ ] If they included an AWS user name, check to see if their user name is listed in [iam_users.tf](https://github.com/department-of-veterans-affairs/devops/blob/master/terraform/environments/global/iam_users.tf)
- [x] Comment in this issue saying which roster the user is listed in and confirming their AWS user name (if applicable)

**Comment by bdech** [comment]
Found on VFS Team Roster

**Comment by bdech** [comment]
@rockwellwindsor-va You've been granted access to `dev`, `staging`, and `sandbox` environments.

Here is some additional documentation for the Vets [API on EKS](https://depo-platform-documentation.scrollhelp.site/developer-docs/vets-api-on-eks#VetsAPIonEKS-TerminalAccess)

In the meantime we will wait on your AWS request for Prod access

**Comment by rockwellwindsor-va** [comment]
sounds good, thanks for the link

**Comment by BillChapmanUSDS** [comment]
We are beginning the process of requiring more specific descriptions of production access and will have a stricter burden of justification.  I will allow the standard 3 months access but depending on the justification the access may be limited in future requests.

**Comment by jknipes** [comment]
@rockwellwindsor-va you have been granted `prod` access for 3 months

**Comment by jknipes** [comment]
https://dsva.slack.com/archives/C02RLAUM2LV/p1688407981007809

**Comment by rockwellwindsor-va** [comment]
Thanks! @jknipes  That link is blocked for me, can you tell me what channel that is in? I am in the process of trying to get access to these but without being able to view any channels it has been tough.

**Comment by jknipes** [comment]
@rockwellwindsor-va that is a link for the Support Team to remind us to remove access to `prod` on Oct 3. 
The support Slack channel for users is #vfs-platform-support

**Comment by rockwellwindsor-va** [comment]
oh my bad, apologies, thanks again for the update

**Comment by davidpetras** [comment]
@rockwellwindsor-va  Your Vets API Terminal Production access has expired.

Would you like to request an extension? You have 72 hours to respond, or your 
access will be revoked and you will need to fill out a new request.

Thank you!

**Comment by rockwellwindsor-va** [comment]
@davidpetras  I would like to request an extension, thank you!

**Comment by davidpetras** [comment]
@LindseySaari @ph-One @ericboehs can one of you please give us the go ahead to extend their access?

**Comment by davidpetras** [comment]
Slack reminder [here](https://dsva.slack.com/archives/C06CY71PCAX/p1716225846845129)

**Comment by lindsay-insco** [comment]
Access revoked

**Comment by rockwellwindsor-va** [comment]
@bdech Apologies I missed this tag.  I will need to renew this, does that require another ticket or can we open this back up?

Again, apologies for missing this tag.
